lib/ansible/plugins/netconf/iosxr.py

```diff
@@ -125,56 +125,80 @@ def guess_network_os(obj):
 
     # TODO: change .xml to .data_xml, when ncclient supports data_xml on all platforms
     @ensure_connected
-    def get(self, filter=None):
+    def get(self, filter=None, remove_ns=False):
```

Comment:
iirc this logic was added to iosxr plugin so that parsing of XML response string using lxml library in the iosxr declarative modules is easier. If the namespace is present in the XML string the XPath traversal in declarative modules requires adding nested namespaces to traversal logic thus making it complex.